### DISCUSSION TITLE
mac编译失败

### DISCUSSION THREAD

**Discussion by cczaqq** [alibaba/xquic#295] [General]
Hello，
     按照文档的步骤：
git submodule update --init --recursive
mkdir build; cd build
cmake -DPLATFORM=mac ..
make -j

编译MAC端xquic失败，报错如下：
Users/cccc/Downloads/project/ThirdParty/xquic/xquic/src/tls/xqc_crypto.h:9:10: fatal error: 'openssl/ssl.h' file not found
/Users/cccc/Downloads/project/ThirdParty/xquic/xquic/src/tls/babassl/xqc_ssl_if.c:5:10: fatal error: 'openssl/ssl.h' file not found
#include <openssl/ssl.h>
         ^~~~~~~~~~~~~~~
#include <openssl/ssl.h>
         ^~~~~~~~~~~~~~~
/Users/cccc/Downloads/project/ThirdParty/xquic/xquic/src/transport/xqc_packet_parser.c:6:10: fatal error: 'openssl/hmac.h' file not found
#include <openssl/hmac.h>
         ^~~~~~~~~~~~~~~~
In file included from /Users/cccc/Downloads/project/ThirdParty/xquic/xquic/src/tls/xqc_hkdf.c:5:
In file included from /Users/cccc/Downloads/project/ThirdParty/xquic/xquic/src/tls/xqc_hkdf.h:8:
/Users/cccc/Downloads/project/ThirdParty/xquic/xquic/src/tls/xqc_crypto.h:9:10: fatal error: 'openssl/ssl.h' file not found
#include <openssl/ssl.h>
         ^~~~~~~~~~~~~~~

找不到openssl，请问该如何解决？

我的环境：MacOS 13.0，
xquic分支: main
commit id: 9f17e546bcc4e0b98d9df60c2b0c98f781e3f7ac

谢谢！

**Comment by Abigail1985** [comment]
我在CMakeList的include_directories里添加openssl路径/opt/homebrew/opt/openssl@3/include 之后解决了'openssl/ssl.h' file not found的问题，但是开始报别的错
